src/windows/Button.tsx

```diff
@@ -217,14 +228,23 @@ export class Button extends ButtonBase implements React.ChildContextProvider<But
         return this.props.tabIndex || 0;
     }
 
-    updateNativeTabIndex(): void {
+    getImportantForAccessibility(): string | undefined {
+        // Focus Manager may override this
+        // We force a default of YES if no property is provided, consistent with the base class
+        return AccessibilityUtil.importantForAccessibilityToString(this.props.importantForAccessibility,
+            Types.ImportantForAccessibility.Yes);
+    }
+
+    updateNativeTabIndexAndIFA(): void {
```

Comment:
What is "IFA"? I'm not familiar with that abbreviation. Could use a comment.

Reply:
IFA == "importantForAccessibility". I think I'll rename everything to "updateNativeAccessibilityProps".